From 40487438af0f2b1785da528218564611ed185706 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Mon, 2 Dec 2024 17:30:10 +0100
Subject: [PATCH 1/3] arm64: dts: renesas: Add R8A779G3 SoC support

Add support for the Renesas R-Car V4H ES3.0 (R8A779G3) SoC, which is an
updated version of the R-Car V4H (R8A779G0) SoC.

Backported from Linux kernel commit
f7cd4b412020 ("arm64: dts: renesas: Add R8A779G3 SoC support")

Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
Reviewed-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Link: https://lore.kernel.org/978c41f932aa2dccd46ad91fc1ddfabacd1c254c.1733156661.git.geert+renesas@glider.be
---
 dts/upstream/src/arm64/renesas/r8a779g3.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 dts/upstream/src/arm64/renesas/r8a779g3.dtsi

diff --git a/dts/upstream/src/arm64/renesas/r8a779g3.dtsi b/dts/upstream/src/arm64/renesas/r8a779g3.dtsi
new file mode 100644
index 00000000000..0295858a026
--- /dev/null
+++ b/dts/upstream/src/arm64/renesas/r8a779g3.dtsi
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/*
+ * Device Tree Source for the R-Car V4H (R8A779G3) SoC
+ *
+ * Copyright (C) 2024 Glider bv
+ */
+
+#include "r8a779g0.dtsi"
+
+/ {
+	compatible = "renesas,r8a779g3", "renesas,r8a779g0";
+};
-- 
2.34.1

